src/models/subscription.ts

```diff
@@ -254,13 +256,18 @@ Subscription.init({
 	gitHubAppId: {
 		type: DataTypes.INTEGER,
 		allowNull: true
+	},
+	avatarUrl: {
```

Comment:
Please remove the change in `subscription.ts`. I mean just have the `db/migrations/20230711040027-add-avatar-to-subscription.js` in this PR. 

As when this code merged in, the `avatarUrl: { type: STRING, allowNull }` will kick off, and result in an db error during runtime.